### ISSUE TITLE
Screen brightness stays at 100% after using fingerprint sensor.

### ISSUE THREAD

**Issue by meehawo** [nanoandrew4/lineage-umi-vanilla#3]
When not using automatic screen brightness mode after using fingerprint sensor (when unlocking phone or using biometric verification inside an app) screen brightness changes to 100% and brightness slider stays at it's previously set position.  

When using automatic screen brightness mode after using fingerprint sensor brightness gets lower with a lag.

Phone: Xiaomi Mi 10 
Build Version: 20240201

**Comment by nanoandrew4** [comment]
@meehawo The fingerprint sensor is the first thing I will look into when I have time, it is an issue with the kernel unfortunately. My options are try to find another kernel that compiles with my sources, which I have tried and failed at multiple times, and to try to fix the kernel I am using right now, which is a daunting task given the immensity of the kernel and the fact that I know nothing about kernel dev. 

The sm8250 devs kernel seemed promising and I got a version working with this issue corrected, but there were other lockscreen bugs that I could not work around, so I have not found a solution that solves this while leaving the rest of the system stable as of yet.

**Comment by Miaoshroom** [comment]
have the latest release fixed the brightness bug?

**Comment by nanoandrew4** [comment]
No, and it will not be fixed for the current versions of LOS I have released, as I cannot change the kernel without risking bricking the phones of anyone who updates from an older version, and trying to solve the issue at the kernel level is beyond me right now. Hopefully I will have some time in the coming months (once other things settle down) to start working on LOS21 and I can pick a kernel that doesn't have the issue or try to work around it there.
